src/app/features/suggested-first-steps/suggested-first-steps.layout.tsx

```diff
@@ -0,0 +1,147 @@
+        justifyContent={['start', 'center']}
+        mt={['loose', 'extra-loose']}
+        pb={['loose', '48px']}
+        width="100%"
+      >
+        <Grid
+          gap={[3, 5, 'unset']}
+          maxWidth={['unset', HOME_FULL_PAGE_MAX_WIDTH]}
+          px={['base-loose', 'base-loose', 'base-loose', 'unset']}
+          templateColumns={['repeat(2, 2fr)', 'repeat(2, 2fr)', 'repeat(4, 1fr)']}
+          width="100%"
+        >
+          <SuggestedFirstStep
+            action="View secret key"
+            body="Don't lose access to your account and crypto"
+            imageFull={BackUpSecretKeyFull}
+            imageFullDone={BackUpSecretKeyFullDone}
+            imagePopup={BackUpSecretKeyPopup}
+            imagePopupDone={BackUpSecretKeyPopupDone}
+            isDone={
+              stepsStatus[SuggestedFirstSteps.BackUpSecretKey] === SuggestedFirstStepStatus.Done
+            }
+            key={SuggestedFirstSteps.BackUpSecretKey}
+            onClick={() =>
+              onSelectStep(RouteUrls.ViewSecretKey, SuggestedFirstSteps.BackUpSecretKey)
+            }
+            title="Back up secret key"
+          />
+          <SuggestedFirstStep
+            action="Get STX"
+            body="Get some STX so you can start using apps"
+            imageFull={AddFundsFull}
+            imageFullDone={AddFundsFullDone}
+            imagePopup={AddFundsPopup}
+            imagePopupDone={AddFundsPopupDone}
+            isDone={stepsStatus[SuggestedFirstSteps.AddFunds] === SuggestedFirstStepStatus.Done}
+            key={SuggestedFirstSteps.AddFunds}
+            onClick={() => onSelectStep(RouteUrls.Fund, SuggestedFirstSteps.AddFunds)}
+            title="Add funds"
+          />
+          <SuggestedFirstStep
+            action="Find apps"
+            body="Try Stacks apps for finance, NFTs, blogging and more"
+            imageFull={ExploreAppsFull}
+            imageFullDone={ExploreAppsFullDone}
+            imagePopup={ExploreAppsPopup}
+            imagePopupDone={ExploreAppsPopupDone}
+            isDone={stepsStatus[SuggestedFirstSteps.ExploreApps] === SuggestedFirstStepStatus.Done}
+            isExternalRoute
+            key={SuggestedFirstSteps.ExploreApps}
+            onClick={() => onSelectStep(exploreAppsExternalRoute, SuggestedFirstSteps.ExploreApps)}
+            title="Explore apps"
+          />
+          <SuggestedFirstStep
+            action="Find NFT"
+            body="Collect and trade NFTs secured by Bitcoin"
+            imageFull={BuyNftFull}
+            imageFullDone={BuyNftFullDone}
+            imagePopup={BuyNftPopup}
+            imagePopupDone={BuyNftPopupDone}
+            isDone={stepsStatus[SuggestedFirstSteps.BuyNft] === SuggestedFirstStepStatus.Done}
+            isExternalRoute
+            key={SuggestedFirstSteps.BuyNft}
+            onClick={() => onSelectStep(buyNftExternalRoute, SuggestedFirstSteps.BuyNft)}
+            title="Buy an NFT"
+          />
```

Comment:
I'd consider all of this content, rather than layout. Could this be passed in as children to the layout component without breaking the styles?

Reply:
Yep, makes sense, let me change.